src/Evolution/Executables/Burgers/EvolveBurgers.hpp

```diff
@@ -158,6 +159,9 @@ struct EvolutionMetavars {
   using observed_reduction_data_tags = observers::collect_reduction_data_tags<
       typename Event<events>::creatable_classes>;
 
+  using factory_classes = tmpl::map<
+      tmpl::pair<StepController, tmpl::list<StepControllers::BinaryFraction>>>;
```

Comment:
Question: Do you think it makes sense to define the list of standard step controllers in a `Factory.hpp`, just so we don't have to edit all executables when adding a new step controller? One can still add extra experimental step controllers with a `tmpl::push_back` in a particular executable.